mock_service/doc/concept.md

```diff
@@ -0,0 +1,198 @@
+# Concept
+## Introduction
+
+As VApp developer I want to have a mock environment where all VSS data points are available and in which actuators will respond when they are requested to change their value.
+
+**Ideally, I should not care about data point availability during development time.**
+
+## Scope
+
+The scope of the Vehicle Mock is a CLI application without a GUI and without phyiscal simulation. It shall be usable to
+
+* configure the mock with expected actuator and sensor behavior to develop an application against
+* configure the mock with actions and reactions to use for integration tests
+
+## Functional requirements
+
+| id | version | description |
+|:---|:-------:|:-----------|
+| #F1  | MVP     | Vehicle Mock shall connect to *Kuksa Data Broker* via gRPC and act as a data provider.
```

Comment:
Which of the requirements listed here is implemented by this PR. All marked as "MVP"?

Reply:
All of the ones marked as MVP **should** implemented by this PR